fbw-a32nx/src/systems/instruments/src/RMP/Components/VhfRadioPanel.tsx

```diff
@@ -56,11 +64,45 @@ const useStandbyVhfFrequency = (side: string, transceiver: number) => {
  * Renders active frequency RadioPanelDisplay and appropriate StandbyFrequency sub-components.
  */
 export const VhfRadioPanel = (props: Props) => {
+  const [_subscriber, setSubscriber] = useState<EventSubscriber<AtcRmpMessages> | null>(null);
   const [active, setActive] = useActiveVhfFrequency(props.vhf);
   const [standby, setStandby] = useStandbyVhfFrequency(props.side, props.vhf);
+  const [atsuFrequency, setAtsuFrequency] = useState<Arinc429Register>(Arinc429Register.empty());
   const [, setValueOppositePanelStandby] =
     props.side === 'L' ? useStandbyVhfFrequency('R', 3) : useStandbyVhfFrequency('L', 3);
 
+  useEffect(() => {
+    const eventBus = new EventBus();
+
+    const subscriber = eventBus.getSubscriber<AtcRmpMessages>();
+
+    subscriber.on('atcHandoverFrequency').handle((frequency: Arinc429Register) => {
+      setAtsuFrequency(frequency);
+    });
+
+    setSubscriber(subscriber);
+  }, []);
```

Comment:
Change to LVar as discussed, then we don't need to pull the avionics framework into the RMP and it's easier to debug the frequency word (+ future study app for A429 words can read it).